helpers/tools.ts

```diff
@@ -170,7 +170,7 @@ For better results, you can specify the region parameter to get results from a s
     dependencies: [
       {
         name: "e2b_code_interpreter",
-        version: "^0.0.11b38",
+        version: "0.0.11b38",
```

Comment:
The latest version is not working so i created a new ticket for the upgrade: https://github.com/run-llama/create-llama/issues/388